games/static/games/proto.js

```diff
@@ -0,0 +1,97 @@
+class Square{
+    constructor(indexOnBoard, div) {
+        this.index = indexOnBoard
+        this.div = div
+    }
+    setCharacter(character) {
+        this.character = character
+        this.div.class = character.getCssClass
+    }
+    isTaken() {}    
+}
+
+function createSquare(indexOnBoard, imageCssClass) {
```

Comment:
didnt you want this to be in board and add the square to its parent?
beacuse like that it does nothing special and you can just use new Sqaure instead of using it

Reply:
yeah' this is from before i will erase it